pkg/cmd/pulumi/login.go

```diff
@@ -145,6 +159,8 @@ func newLoginCmd() *cobra.Command {
 	}
 
 	cmd.PersistentFlags().StringVarP(&cloudURL, "cloud-url", "c", "", "A cloud URL to log in to")
+	cmd.PersistentFlags().StringVar(&defaultOrg, "org-name", "", "A default org to associate with the login. "+
```

Comment:
Apologies if this has already been discussed, but I am wondering if this should be named `default-org`?

```
pulumi login --default-org demo-org
```

Or maybe even just `org`?

```
pulumi login --org demo-org
```

Reply:
I like the rename to `default-org` as it's clearer